### ISSUE TITLE
Sit on gameobject = Crash (BB #499)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#499]
This issue was migrated from bitbucket.
**Original Reporter:** 
**Original Date:** 24.02.2014 18:00:58 GMT+0000
**Original Priority:** critical
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/499

<hr>

Last rev from today. Compiled without problems some created characters crash server when sit on chair / bench. 

kernel: [3700978.994023] oregon-core[19399]: segfault at 0 ip 000000000094a448 sp 00007fff6e594d30 error 4 in oregon-core6[400000+a04000]

Nothing written in logs(full debug logging turned on and nothing)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 24.02.2014 18:03:16 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/499/#comment-8572237

<hr>

Can I have the crash log? It's probably located in a folder called Crashes. It'll have todays date on it and it'll have a .txt version and a dump version. I'll take the txt if it's not a problem.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 24.02.2014 18:06:34 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/499/#comment-8572287

<hr>

And let me ask you this, does it happen when they've been drinking? (Drunk in game :))

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 24.02.2014 18:13:20 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/499/#comment-8572378

<hr>

I've figured it out, fix will be up in about 10 minutes :)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 24.02.2014 19:11:46 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/499/#comment-8573523

<hr>

Well, I'll be damned. My hard drive failed.

```
#!c++

   if (FactionTemplateEntry const* faction = sFactionTemplateStore.LookupEntry(obj->GetGOInfo()->faction))
    if (!_player->isGameMaster() &&
        !faction->IsFriendlyTo(*sFactionTemplateStore.LookupEntry(_player->getFaction())) &&
```

Try that, and let me know

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 24.02.2014 20:58:30 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/499/#comment-8575204

<hr>

Resolved in 81f0a99.
Update your core, recompile and enjoy the game :)
Thank you for the report.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** smoldar
**Original Date:** 25.02.2014 10:23:26 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/499/#comment-8597398

<hr>

Tested now is working fine. :-) thank
